tianshou/highlevel/experiment.py

```diff
@@ -443,18 +544,15 @@ def with_optim_factory(self, optim_factory: OptimizerFactory) -> Self:
 
     def with_optim_factory_default(
         self,
-        betas: tuple[float, float] = (0.9, 0.999),
-        eps: float = 1e-08,
-        weight_decay: float = 0,
+        **kwargs: Unpack[DefaultOptimFactoryParams],
     ) -> Self:
-        """Configures the use of the default optimizer, Adam, with the given parameters.
+        """Configures the use of the default optimizer, with the given parameters.
 
-        :param betas: coefficients used for computing running averages of gradient and its square
-        :param eps: term added to the denominator to improve numerical stability
-        :param weight_decay: weight decay (L2 penalty)
+        :param kwargs: the parameters to use for the optimizer, see `DefaultOptimFactoryParams`.
         :return: the builder
         """
-        self._optim_factory = OptimizerFactoryAdam(betas=betas, eps=eps, weight_decay=weight_decay)
```

Comment:
How is this an improvement?
It seems to me that this just creates confusion as to what the default optimiser and its arguments are.